### ISSUE TITLE
could you please provide the weights of VideoMAEv2 pre-trained on Kinetics-400?

### ISSUE THREAD

**Issue by fmthoker** [OpenGVLab/VideoMAEv2#25]
Hi,
Thanks for sharing this great work. I am doing an evaluation of VIT models on specific video benchmarks and would like to include videomaev2.  To make the comparison fair with other video SSL works, I need the weights of the ViT-B model pretrained on the Kinetics-400 dataset with the videomae2 loss function. Is it possible to share the corresponding weights? My email is fmthoker@gmail.com

**Comment by congee524** [comment]
Hi,
VideoMAE v2 uses the same loss function MSE Loss as v1. We also only pre-trained on the UnlabeledHybrid dataset, there's no version trained on k400.
